.github/workflows/slack.yml

```diff
@@ -0,0 +1,18 @@
+name: Slack
+on:
+  workflow_run:
+    workflows: [CI, Update repos]
+    types: [completed]
+    branches: [main]
+jobs:
+  on-failure:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      - name: Post to Slack
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+          channel-id: 'C4K6M7P5E'
```

Comment:
Is this #prod-general? It might become a bit too noisy but if we move it to some other channel we might ignore them instead. We'll have to find the balance.